assets/setup-wizard/features/feature-description.js

```diff
@@ -1,34 +1,64 @@
-import { __ } from '@wordpress/i18n';
+import { sprintf, __ } from '@wordpress/i18n';
 import { logLink } from '../log-event';
 
+/**
+ * @typedef  {Object} Feature
+ * @property {string} rawTitle Feature title.
+ */
 /**
  * Feature description component
  *
- * @param {Object} props
- * @param {string} props.slug    Feature slug.
- * @param {string} props.excerpt Feature excerpt.
- * @param {string} [props.link]  Feature link.
+ * @param {Object}    props
+ * @param {string}    props.slug               Feature slug.
+ * @param {string}    props.excerpt            Feature excerpt.
+ * @param {string}    [props.link]             Feature link.
+ * @param {Feature[]} [props.selectedFeatures] Feature link.
  */
-const FeatureDescription = ( { slug, excerpt, link } ) => (
-	<>
-		{ excerpt }
-		{ link && (
-			<>
-				{ ' ' }
-				<a
-					className="sensei-setup-wizard__learn-more link__color-primary"
-					href={ link }
-					target="_blank"
-					rel="noopener noreferrer"
-					{ ...logLink( 'setup_wizard_features_learn_more', {
-						slug,
-					} ) }
-				>
-					{ __( 'Learn more', 'sensei-lms' ) }
-				</a>
-			</>
-		) }
-	</>
-);
+const FeatureDescription = ( { slug, excerpt, link, selectedFeatures } ) => {
+	let wcObservation = null;
+
+	if ( 'woocommerce' === slug && selectedFeatures ) {
```

Comment:
I'm not sure about passing the selected features list to this component, ideally it should only need info on the one it's displaying. Maybe we could calculate this in somewhere like the `useEffect` hook where the WooCommerce feature is added and add it as a `notice` property to the feature object? WDYT?

Reply:
WDYT of this approach? https://github.com/Automattic/sensei/pull/3297/commits/bbf674644ce6acdb314fed6f475bd196dec0e1a7

Reply:
Looks a bit cleaner 🛀